blog/2025-02-16-new-config-entry-states.md

```diff
@@ -0,0 +1,72 @@
+---
+author: Erik Montnemery
+authorURL: https://github.com/emontnemery
+title: "Changed config entry state transitions"
+---
+
+Config entry state transitions when unloading and removing entries has been modified:
+
+- A new state `ConfigEntryState.UNLOAD_IN_PROGRESS` is added, which is set before the integration's `async_unload_entry` is called
+  Rationale:
```

Comment:
```suggestion
- A new state `ConfigEntryState.UNLOAD_IN_PROGRESS` is added, which is set before the integration's `async_unload_entry` is called. Rationale:
```
"Rationale" is not rendered on a separate line, since this is a bullet point list

Reply:
I intended for the line break though. I've updated to add a `<br>` tag.

Reply:
Oh yeah, `<br>` works too!